src/Microsoft.AspNet.Server.Kestrel/App.config

```diff
@@ -0,0 +1,4 @@
+﻿<configuration>
+  <dllmap dll="libuv.dll" target="libuv.so.1" os="linux"/>
+  <dllmap dll="libuv.dll" target="native/darwin/universal/libuv.dylib" os="osx"/>
```

Comment:
How does the `app.config` get used? Normally I thought only EXE's can have a config file, no?